### PR TITLE
Make edit button dark blue

### PR DESCRIPTION
**Pull request by openjck**
A recent A/B test showed that around 31% more people click the edit
button when it's a darker shade of blue.

Full results: http://optimize.ly/~KSOEmr?token=63f0bef5c746e00caa31#view=2
Analysis: https://groups.google.com/d/msg/mozilla.dev.mdn/98GP_Vh30Cc/GSL_L-f-pJYJ

### CODE REVIEW

File: media/stylus/components/wiki/page-buttons.styl [hunk 1]
```diff
@@ -34,6 +34,13 @@
     }
 }
 
+/* Users can style zone landing pages. We can't be sure that dark blue will look
+   good on background colors they choose. */
+body:not(.zone-landing) #edit-button {
```
Comment (darkwing) [inline media/stylus/components/wiki/page-buttons.styl]: `:not` sometimes worries me.  Thoughts @stephaniehobson ?

Comment (openjck) [inline media/stylus/components/wiki/page-buttons.styl]: https://twitter.com/chriscoyier/status/414149535800127488 :smile:

If measurement shows that this noticeably affects performance, though, we should address it.

Comment (darkwing) [inline media/stylus/components/wiki/page-buttons.styl]: Not from a perf standpoint, from a maintenance/cleanliness standpoint.

Comment (openjck) [inline media/stylus/components/wiki/page-buttons.styl]: Ah. Gotcha.

I spent some time thinking about that. We could alternatively give non-landing pages a new class. Something like `.standard-article`. But then if we create a third category of page later on, selecting `.standard-article` won't be enough.

We ultimately want to select everything that *isn't* a zone landing page, so I think the `:not` selector is a natural fit.

Comment (stephaniehobson) [inline media/stylus/components/wiki/page-buttons.styl]: I think it makes the code more readable. I'm okay with it.

**Comment by stephaniehobson** [comment]
:+1:

**Pull request merged**